rpa-builder/6.2/modules/ROOT/pages/toolbox-system-write-bot-message.adoc

```diff
@@ -0,0 +1,26 @@
+= Write Bot Message 
+
+The Write Bot Message Action Step enables you to enrich the run result message the RPA Bot generates after each run. This message shows in the run results page of RPA Manager. 
```

Comment:
This sentence seems weird. Do you mean?:

- "This message shows on the run-results page of RPA Manager". Is there a page named "run results"?
- "This message shows results on the page of RPA Manager on the run". The results are displayed as the manager runs.